### PR TITLE
Issue#6 Connect server with client And Make map feature

### PR DESCRIPTION
**Pull request by chickengak**
#6

### CODE REVIEW

File: fronted/main.html [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    <style>
+      html, body {
+        height: 100%;
+        margin: 0;
+        padding: 0;
+      }
+    </style>
+    <link rel="stylesheet" href="main.css" />
+    <script>
+      var map;
```
Comment (Jelldo) [inline fronted/main.html]: change var type to 'let'

File: fronted/main.html [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+      var geocoder;
+      var latlng;
+      var initLocation = {lat:37.570204, lng: 126.976956}; // 서울 중심 좌표: 37.570204 , 126.976956
+      function initMap() {
```
Comment (Jelldo) [inline fronted/main.html]: initializeMap

File: frontend/geocoding.js [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+    //,disableDefaultUI: true,zoomControl: false,
+    scaleControl: true
+  });
+  geocoder = new google.maps.Geocoder();
+  setupEvents();
+  centerChanged();
+}
+function setupEvents() {
+  reverseGeocodedLast = new Date();
+  centerChangedLast = new Date();
```
Comment (GodYoungHwan) [inline frontend/geocoding.js]: delete

File: frontend/main.html [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+    <script src="geocoding.js"></script>
+  </head>
+  <body>
+    <div id ="search">
+      Search: <input type="text" id="address"/>
+              <input type="button" value="Go" onclick="geocode()">
+      <a id = "search-latlng">Lat/Lng: <label id="latlng">(37.570204, 126.976956) </a>
+    </div>
+    <div id="map"></div>
+<!--API 설정 안내
```
Comment (GodYoungHwan) [inline frontend/main.html]: overlap code

**Pull request merged**